.github/workflows/pypi_publish.yml

```diff
@@ -30,7 +30,7 @@ on:
         description: "Update Connector Builder. If true, the workflow will create a PR to bump the CDK version used by Connector Builder."
         type: boolean
         required: true
-        default: true
+        default: false
```

Comment:
I changed this to default to "false" after I accidentally left it selected for my pre-release test this morning.

Reply:
Good catch!